arch/arm64/start.S

```diff
@@ -35,6 +35,10 @@ arm_reset:
     /* if we came in at higher than EL1, drop down to EL1 */
     bl      arm64_elX_to_el1
 
+    /* disable EL1 FPU traps */
+    mov     x0, #(0b11<<20)
```

Comment:
Looks good, can you switch this to using the 'tmp' register however? Reason is in some situations x0 may still contain an argument from the bootloader or from PSCI when booting secondary cpus. The registers in most of this file try to avoid using x0-x3 or so.